tests/test_schema.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+import jsonschema
+import pyarrow as pa
+import pytest
+
+from meds import patient_schema, label, dataset_metadata
+
+
+def test_patient_schema():
+    """
+    Test that mock patient data follows the patient_schema schema.
+    """
+    patient_data = {
+        "patient_id": [123],
+        "events": [[{  # Nested list for events
+            "time": datetime.datetime(2020, 1, 1, 12, 0, 0),
+            "measurements": [{  # Nested list for measurements
+                "code": "some_code",
+                "text_value": "Example",
+                "numeric_value": 10.0,
+                "datetime_value": datetime.datetime(2020, 1, 1, 12, 0, 0),
+                "metadata": None
+            }]
+        }]]
+    }
+
+    patient_table = pa.Table.from_pydict(patient_data, schema=patient_schema())
```

Comment:
Can you swap this to from_pylist?

It's much easier to understand for this sort of data, and we should push people to using from_pylist over from_pydict.

Reply:
See https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.from_pylist

Reply:
Thanks, switched `from_pydict` to `from_pylist` as suggested.